src/LibSingular.jl

```diff
@@ -2,7 +2,13 @@ module libSingular
 
 import Libdl
 using CxxWrap
-@wrapmodule(realpath(joinpath(@__DIR__, "..", "deps", "usr", "lib", "libsingularwrap." * Libdl.dlext)))
+
+const libsingularwrap_path = joinpath(@__DIR__, "..", "deps", "usr",
+                "lib", "libsingularwrap." * Libdl.dlext)
+if !isfile(libsingularwrap_path)
+    error("""Singular.jl needs to be compiled; please run `using Pkg; Pkg.build("Polymake")`""")
```

Comment:
the error message should probably be: Pkg.build("Singular")

Reply:
Oops, good catch -- fixed now